diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index 7149c64..9573c66 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -685,12 +685,7 @@ if (__DEV__) {
   Object.defineProperty(module.exports, 'DatePickerAndroid', {
     configurable: true,
     get() {
-      invariant(
-        false,
-        'DatePickerAndroid has been removed from React Native. ' +
-          "It can now be installed and imported from '@react-native-community/datetimepicker' instead of 'react-native'. " +
-          'See https://github.com/react-native-datetimepicker/datetimepicker',
-      );
+      return require('@react-native-picker/picker');
     },
   });
   /* $FlowFixMe[prop-missing] This is intentional: Flow will error when
